### PR TITLE
Update snapshot version to 3.0.3

### PR DESCRIPTION
**Pull request by xing-yang**
Updates snapshot version to 3.0.3 and also updates owners file.

### CODE REVIEW

**Comment by xing-yang** [comment]
/hold

Wait for snapshot 3.0.3 to be released first.

**Comment by xing-yang** [comment]
/hold cancel

snapshot 3.0.3 images are available now.

File: config/jobs/kubernetes-csi/OWNERS [hunk 1]
```diff
@@ -3,8 +3,10 @@
 reviewers:
 - msau42
 - saad-ali
+- xing-yang
```
Comment (msau42) [inline config/jobs/kubernetes-csi/OWNERS:6]: As followup, @jsafrane and @pohly should be added to approvers as well

**Review by msau42 — COMMENTED** [review]
/lgtm
/approve

File: config/jobs/kubernetes-csi/OWNERS (hunk 1 above)
Comment (xing-yang) [reply config/jobs/kubernetes-csi/OWNERS:6]: sure.

**Pull request merged**